client/src/resources/transaction.rs

```diff
@@ -1,11 +1,26 @@
 use chrono::prelude::*;
 use super::{deserialize, amount::Amount};
 
+/// Memos are a useful source for adding meta data to a transaction.
+/// A consists of a type and content (unless memo type is none).
+///
+/// To learn more about the concept of memos in the Stellar network, take a look at the Stellar memo description here:
+/// https://www.stellar.org/developers/guides/concepts/transactions.html#memo
```

Comment:
I think if you wrap the url in <> it will auto-link in the docs.

Reply:
fixed!